task/02-numbers-tasks.js

```diff
@@ -73,7 +71,9 @@ function getAverage(value1, value2) {
  *   (-5,0) (10,-10) => 18.027756377319946
  */
 function getDistanceBetweenPoints(x1, y1, x2, y2) {
-    throw new Error('Not implemented');
+  let a = Math.abs(x1 - x2);
+  let b = Math.abs(y1 - y2);
+  return Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
```

Comment:
JFYI: calculate value of expressions, save it to variable and return variable. Intel/Google styleguide.